action.yml

```diff
@@ -35,7 +35,7 @@ runs:
       BASE: ${{ inputs.base }}
       HEAD: ${{ inputs.head }}
       ARGS: ${{ inputs.extra_args }}
-      COMMITS: ${{ toJson(github.event.commits) }}
```

Comment:
@sryabkov removing this might break the action for some users

Reply:
@ zricethezav Can you elaborate and give an example?

Reply:
> Maybe I am missing something, but the way I am looking at things, this variable is only set for the one step and it is not available outside of it, so it is not usable by anything other than the code in that step... If there is something I am missing, I am very curious...

You're not missing anything, I was lacking sleep. With slightly better rested eyes I now see this won't break anything for the reasons you explained. 

Changes LGTM 👍🏻

Reply:
Appreciate the thorough descriptions and clarifying comments. Top notch PR 🤌